misc/scripts/png2c.py

```diff
@@ -20,7 +20,7 @@
   -s    embed the image size in the image names in generated code."""
 
 if len(sys.argv) < 2:
-        print USAGE
+        print( USAGE )
```

Comment:
Stylistic comment - I would get rid of the extra spaces you have around the arguments in all of these calls.  It's not typically how Python is stylized and not consistent with the rest of the file.

Reply:
Fixed with the latest commit.

Reply:
LGTM.

Reply:
@swt2c, thx.